tests/test_user.py

```diff
@@ -809,27 +812,24 @@ def test_deep_tree_check_role(self):
         for perm in ('view', 'edit'):
             for patient in (
                 patient_w_id, patient_x_id, patient_y_id, patient_z_id):
```

Comment:
This is an example of a good test to parameterize, you could pass each patient as a parameter FYI

Reply:
Gotcha